packages/Algernon.package/ALGWorldMenuType.class/class/filterMenuItems..st

```diff
@@ -0,0 +1,4 @@
+helpers
+filterMenuItems: aMorph
```

Comment:
hm, there isn't really any filtering of menu items going on, is there?

what about `scrapeMenuItems`? I fell like "filter" / "select" reads as if the `aMorph` was a collection, though it isn't (and the name `each` doesn't make that clear either)

Reply:
Yeah, you're right. I would suggest sth like `extractEntriesFromMenuItem` as we do exactly that, no?